lib/headRoute.js

```diff
@@ -0,0 +1,30 @@
+function headRouteOnSendHandler (req, reply, payload, done) {
+  // If payload is undefined
+  if (payload === undefined) {
+    reply.header('content-length', '0')
+    return done(null, null)
+  }
+
+  // If is a stream, we remove content-type header without appending
+  // content-length one, and resume it right away
```

Comment:
why we need to remove the content-type?

Reply:
To follow the same implementation as `express`, as discussed [here](https://github.com/fastify/fastify/issues/2619#issuecomment-720340038). But checking twice, if the `custom-header` exists because of the implementation, should we leave or better remove it?

Reply:
I would not remove the content-type. This does not seem correct.

Reply:
Sounds great, done in 08f3772